docs/SPEC.md

```diff
@@ -3685,31 +3685,39 @@ from(bucket: "telegraf/autogen")
+| ----              | ----           | -----------                                                                                                                                                                                                                        |
+| bucket            | string         | Bucket is the bucket name into which data will be written.                                                                                                                                                                         |
+| bucketID          | string         | BucketID is the bucket ID into which data will be written.                                                                                                                                                                         |
+| org               | string         | Org is the organization name of the bucket.                                                                                                                                                                                        |
+| orgID             | string         | OrgID is the organization ID of the bucket.                                                                                                                                                                                        |
+| host              | string         | Host is the location of a remote host to write to. Defaults to `""`.                                                                                                                                                               |
+| token             | string         | Token is the authorization token to use when writing to a remote host. Defaults to `""`.                                                                                                                                           |
+| measurementColumn | string         | MeasurementColumn is the measurement into which data will be written. Defaults to `"_measurement"`.                                                                                                                                |
+| fieldColumn       | string         | FieldColumn defines the column from which the field keys and values will be taken.  Defaults to `"_field"`.                                                                                                                                   |
+| timeColumn        | string         | TimeColumn is the name of the time column of the output.  Defaults to `"_time"`.                                                                                                                                                   |
+| valueColumn       | string         | ValueColumn is the name of the value column of the output.  Defaults to `"_value"`.                                                                                                                                                 |
```

Comment:
so you expect the user to drop incompatible/unwanted columns before writing back?  e.g. we are losing the tagColumns parameter which seems like it could still be useful.